### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by jasonl**

### CODE REVIEW

File: API-Documentation/api.md [hunk 1]
```diff
@@ -20,21 +20,19 @@ All requests to the endpoint must be made over [HTTPS](http://en.wikipedia.org/w
...
 The ChartMogul Metrics API uses [HTTP Basic Authentication](http://en.wikipedia.org/wiki/Basic_access_authentication). This means that with **every** request you must provide your API key in the request header.
 
-**Where's my API key?**
-
-You can find your API key in the ChartMogul [Admin pages](https://app.chartmogul.com/#admin/api) (click the gear icon in the top right of your account - you need to be an admin on your account) - then you'll see an API tab:
+To authenticate with the ChartMogul Metrics API, pass your **Account Token** as your username, and your **Secret Key** as your password for HTTP basic authentication. These two
+tokens can be found in the [Admin pages](https://app.chartmogul.com/#admin/api) section of the ChartMogul web application. You will need to be an administrator on your ChartMogul
+account to see these.
 
 ![Admin - API](admin-api.jpg?raw=true "Admin API")
```
Comment (ytvinay) [inline API-Documentation/api.md]: This image is more accurate given recent changes to the UI - https://github.com/chartmogul/enrichment-api/blob/master/API-Documentation/admin-api.jpg

File: API-Documentation/api.md [hunk 2]
```diff
@@ -20,21 +20,19 @@ All requests to the endpoint must be made over [HTTPS](http://en.wikipedia.org/w
...
-
 ## HTTP Response Codes
 
-The following response codes may be returned by the endpoint:
+The following response codes may be returned by the endpoints:
 
 - `200` - OK (Everything worked as expected)
 - `400` - Bad Request (Often missing a required parameter)
-- `401` - Unauthorized (No valid API key provided in the request header - see Authentication)
+- `401` - Unauthorized (No valid credentials provided - see Authentication)
```
Comment (ytvinay) [inline API-Documentation/api.md]: Link the word [Authentication](https://github.com/chartmogul/metrics-api/blob/master/API-Documentation/api.md#authentication) ?

File: API-Documentation/api.md [hunk 3]
```diff
@@ -433,43 +500,56 @@ curl https://api.chartmogul.com/v1/metrics/mrr-churn-rate \
...
 ```
+
+The `entries` key contains an object for each month, with the following data:
+- `date` - The date of the end of the month
+- `mrr-churn-rate` - The MRR churn rate at the end of that month, expressed as a percentage.
+
+Note that MRR Churn Rate can be both negative, and larger than 100% - in this case, a negative churn rate of -300.0% means that the MRR actually quadrupled during that period.
+Normal values will typically be between -10% and +10%, though that depends, naturally, on what your customers are actually doing.
+
+We have some great "My Other Startup Has Negative Churn" T-shirts here at ChartMogul. Send a message to support if you want one - supplies are limited!
```
Comment (ytvinay) [inline API-Documentation/api.md]: Link the text [Send a message](mailto:support@chartmogul.com) ?

**Comment by ytvinay** [comment]
The Endpoint Overview section needs the HTTP methods included in there. Would be great to link through each endpoint from the Overview section through to the definition like in the [Enrichment API Docs](https://github.com/chartmogul/enrichment-api/blob/master/API-Documentation/api.md#endpoint-overview)

**Comment by ytvinay** [comment]
`GET /customers/{id}/subscriptions` needs its endpoint definition

File: API-Documentation/api.md [hunk 4]
```diff
@@ -271,38 +310,48 @@ curl https://api.chartmogul.com/v1/metrics/arpa \
...
 
 #### Supported Parameters
 
 
- - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
+ - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `2015-05-12`
 
- - `end-date` (Date) (*required*) - The end date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
-
- - `interval` (String) - only `"month"` (default) interval is supported
```
Comment (ytvinay) [inline API-Documentation/api.md]: Why remove this? It's included in the example curl request below

File: API-Documentation/api.md [hunk 5]
```diff
@@ -433,43 +500,56 @@ curl https://api.chartmogul.com/v1/metrics/mrr-churn-rate \
...
 
 #### Supported Parameters
 
+ - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `2015-05-12`
 
- - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
-
- - `end-date` (Date) (*required*) - The end date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
-
- - `interval` (String) - only `"month"` (default) interval is supported
```
Comment (ytvinay) [inline API-Documentation/api.md]: Include an informational note about this maybe?

File: API-Documentation/api.md [hunk 6]
```diff
@@ -384,38 +448,42 @@ curl https://api.chartmogul.com/v1/metrics/customer-count \
...
 
+ - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `2015-05-12`
 
- - `start-date` (Date) (*required*) - The start date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
+ - `end-date` (Date) (*required*) - The end date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `2015-05-12`
 
- - `end-date` (Date) (*required*) - The end date of the required period of data. An [ISO-8601](http://en.wikipedia.org/wiki/ISO_8601) formatted date, e.g. `"2015-05-12"`
+ - `geo` (String) - A comma-separated list of [ISO 3166-1 Alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) formatted country codes to filter the results to, e.g. `US,UK,DE`.
 
- - `interval` (String) - only `"month"` (default) interval is supported
```
Comment (ytvinay) [inline API-Documentation/api.md]: Include an informational note on this maybe?

File: API-Documentation/api.md (hunk 4 above)
Comment (jasonl) [inline API-Documentation/api.md]: I must have missed that. It's useless to specify a parameter that can only be passed one possible value.

File: API-Documentation/api.md (hunk 1 above)
Comment (jasonl) [inline API-Documentation/api.md]: Updated.

File: API-Documentation/api.md (hunk 2 above)
Comment (jasonl) [inline API-Documentation/api.md]: Updated.

File: API-Documentation/api.md (hunk 3 above)
Comment (jasonl) [inline API-Documentation/api.md]: Added.

File: API-Documentation/api.md (hunk 6 above)
Comment (jasonl) [inline API-Documentation/api.md]: Done.

**Comment by jasonl** [comment]
Have added all your suggestions @ytvinay. Can you :+1: this?

File: API-Documentation/api.md [hunk 7]
```diff
@@ -20,62 +20,60 @@ All requests to the endpoint must be made over [HTTPS](http://en.wikipedia.org/w
...
+- [`GET /metrics/mrr`](#get-metricsmrr) - Monthly Recurring Revenue
+- [`GET /metrics/arr`](#get-metricsarr) - Annualized Run Rate
+- [`GET /metrics/arpa`](#get-metricsarpa) - Average Revenue Per Account
+- [`GET /metrics/asp`](#get-metricsasp) - Average Sale Price
+- [`GET /metrics/customer-count`](#get-metricscustomer-count) - Number of customers
+- [`GET /metrics/customer-churn-rate`](#get-metricscustomer-churn-rate) - Customer Churn Rate
+- [`GET /metrics/mrr-churn-rate`](#get-metricsmrr-churn-rate) - MRR Churn Rate
+- [`GET /metrics/ltv`](#get-metricsltv) - Customer Lifetime Value
+- [`GET /customers/search`](#get-customerssearch) - Find information for a specific customer
+- [`GET /customers/{customer_id}/subscriptions`](#get-customerscustomer_idsubscriptions) - Get a list of subscriptions for a single customer
```
Comment (ytvinay) [inline API-Documentation/api.md]: @jasonl This endpoint needs definition. Otherwise, PR looks good.

**Comment by ytvinay** [comment]
Can be merged, provided the following PR is approved for merge as well,

https://github.com/chartmogul/metrics-api/pull/11/

**Pull request merged**